I do magic/promises-callbacks-js/src/html/index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE HTML>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>Super Agentes - Home</title>
+    <!--Bootstrap grid-->
+    <link rel="stylesheet" type="text/css" href="main.css">
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/mustache.js/2.3.2/mustache.min.js"></script>
+
+    <script type="text/x-handlebars-template" id="template">
+      {{#companies}}
+        <div class="agent">
+          <p class="agent__name">{{name}}</p>
+          {{#isCertified}}
+          <img src="images/certified.png" alt="ribbon" class="agent__ribbon">
+          {{/isCertified}}
+          <div class="agent__separator"></div>
+          <img class="agent__photo" src="{{image}}">
+          <div class="agent__description">{{description}}</div>
+          <div class="agent__price">Desde: ${{rate}} / {{hours}} horas</div>
+          <div class="agent__button">
+            <div class="agent__button-text">Contratar</div>
+          </div>
+        </div>
+      {{/companies}}
+    </script>
+
+    <script type="text/javascript" src="js/load-data.js"></script>
+  </head>
+
+  <body language=JavaScript onLoad="getAgents('https://api.myjson.com/bins/uptto')">
+    <!--start header-->
+    <header class="header">
+      <div class="logo"></div>
+      <div class="menu-button">
+        <div class="menu-button__rectangle-top"></div>
+        <div class="menu-button__rectangle-mid"></div>
+        <div class="menu-button__rectangle-bottom"></div>
+      </div>
+      <div class="options">
+        <a class="options__directory" href="">
+          Directorio de Agentes
+          <div class="options__directory--selected"></div>
+        </a>
+        <a class="options__contact" href="">
+          Contacto
+          <div class="options__contact--selected"></div>
+        </a>
+        <a class="options__login" href="">
+          Ingresar
+          <div class="options__login--selected"></div>
+        </a>
+      </div>
+
+    </header>
+    <!--start body content-->
+    <div class="body-content">
+      <div class="image"></div>
+      <div class="title">Los Mejores agentes de seguridad</div>
+      <div class="searchbar">
+          <input type="text" class="searchbar__textfield" placeholder="Buscar por Ubicación" name="Location" onkeypress="validateEnter(event)">
```

Comment:
Same as the next comment; Don't mix JS in HTML; is not a good practice. Use event handlers instead of `onkeypress=(...)`.

Reply:
Done